pages/common/drush.md

```diff
@@ -3,25 +3,17 @@
 > A command-line shell and scripting interface for Drupal.
 > More information: <https://www.drush.org>.
 
-- Download module "foo":
-
-`drush dl {{foo}}`
-
-- Download version 7.x-2.1-beta1 of module "foo":
-
-`drush dl {{foo}}-7.x-2.1-beta1`
-
 - Enable module "foo":
```

Comment:
Does this also install the module if it's not installed already?

Just asking because you've removed the download example here.

Reply:
Also, if the download examples are removed, how does one download a module in drupal 8 ?

Reply:
Drupal 8 modules are downloaded with [Composer](https://getcomposer.org/).

`drush dl` now outputs this:
```
  dl has been deprecated. Please build your site using Composer. Add new projects with composer require drupal/[project-name]. Use https://www.drupal.org/project/composer_generate to build a composer.json which represents the enabled m  
  odules on your site.
```

Reply:
Thanks !